definitions/manifest/deployment.yaml

```diff
@@ -51,18 +51,18 @@ spec:
           args:
           - "--filterK8sResources=[Event,*,*][*,kube-system,*][*,kube-public,*][*,kube-node-lease,*][Node,*,*][APIService,*,*][TokenReview,*,*][SubjectAccessReview,*,*][*,kyverno,*][Binding,*,*][ReplicaSet,*,*][ReportChangeRequest,*,*][ClusterReportChangeRequest,*,*][PolicyReport,*,*][ClusterPolicyReport,*,*]"
           # customize webhook timeout
-          #- "--webhooktimeout=4"
+          #- "--webhookTimeout=4"
           # enable profiling
           # - "--profile"
           # configure the workers for generate controller
-          # - --gen-workers=20
+          # - --genWorkers=20
           - "-v=2"
           ports:
             - containerPort: 9443
               name: https
               protocol: TCP
             - containerPort: 8000
-              name: metrics-port
```

Comment:
Kubernetes does not allow camel case for port names. 

We do not need to change this setting in the YAMLs. The issue is only to make the kyverno flags consistent.